backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -137,23 +78,41 @@ private MimeMessage createEmail(String subject, String bodyText) throws Messagin
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
     email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
-    email.setSubject(subject);
+    email.setSubject(subjectText);
     email.setText(bodyText);
     return email;
   }
 
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName, String firstMatchRecipientEmail)
+      throws MessagingException, IOException {
+    MimeMessage email =
+        createEmail(
+            firstMatchRecipientEmail,
+            "Ad-lib: We found you a match !",
+            String.format(
+                " We found you a match with %s "
+                    + " Check your calendar for your meeting event,"
+                    + " and feel free to join the Meet call now!\n",
+                firstMatchRecipientName));
+    Message messageWithEmail = createMessageWithEmail(email);
+    service.users().messages().send("me", messageWithEmail).execute();
```

Comment:
WDYT about extracting "me" as a string constant with a javadoc so we know what this means and what to replace this with if we ever did get a different authorized sender?

Reply:
That would be useful !